float32 param1
float32 param2
float32 param3
float32 param4
---
string status
bool result